GuessingGame.Tests/UnitTest1.cs

```diff
@@ -7,16 +7,34 @@ namespace GuessingGame.Tests;
 public class UnitTest1
 {
     [Fact]
-    public void GetResult_found()
+    public void Test_For_Winning_GuessingGameResult()
     {
 
-        int minRange = 4;
+        int minRange = 1;
         int maxRange = 5;
         int guessNumber = 3;
+        int amount =12;
         INumberGenerator numberGenerator = Substitute.For<INumberGenerator>();
         numberGenerator.Generate(minRange, maxRange).Returns(guessNumber);
-        Service service = new Service(numberGenerator);
-        string result=service.GetResult(minRange,maxRange,6);
+        GuessingGameService service = new GuessingGameService(numberGenerator);
+        string result = service.GuessingGameResult(minRange, maxRange, 5,amount,3);

```

Comment:
Good work on updating your code to make it more testable, its only a small thing but a good reusage of the guessNumber would be good in the method Invokation as well so the number is consistent if you decided to change the value in future. ie service.GuessingGameResult(minRange, maxRange, 5, amount, guessNumber);